astroplan/constraints.py

```diff
@@ -546,6 +546,48 @@ def __init__(self, min=None, max=None, ephemeris=None):
         self.max = max
         self.ephemeris = ephemeris
 
+    @classmethod
+    def dark(cls, **kwargs):
+        """
+        This will use default values (acceptable fractional illumination)
+        of max=0.25 with no minimum. You can input min= or max= to change
+        the values and ephemeris= if a different ephemeris is desired,
+        by default it uses ``astropy.coordinates.solar_system_ephemeris.set``
```

Comment:
@kvyh While you're at it, it looks like some other docstrings have that double-ticked reference to `solar_system_ephemeris` (e.g. [see here](https://github.com/astropy/astroplan/blob/master/astroplan/constraints.py#L482))). Would you change those to `` `~astropy.coordinates.solar_system_ephemeris.set` `` also?